### PR TITLE
change social inputs in profile

### PR DESCRIPTION
**Pull request by fuentesloic**
according #61 
Result: Neomad need just his/her social username to fill social input in profile
capture d ecran 2017-09-21 a 16 12 52

### CODE REVIEW

**Comment by fuentesloic** [comment]
@newick It's aligned ^^
@vinyll It's working ^^

**Review by vinyll — DISMISSED** [review]
I'm not fully conviced by this approach, but it seems to do the job for now ;)
Let's use that and see with time. 👍

**Comment by newick** [comment]
@fuentesloic facebook icon could be centered on the email one ? :D

**Comment by fuentesloic** [comment]
@newick done about centered FB
+ @vinyll to enhance a little bit I did that:
output_wuwfgq

File: templates/user/profile.html [hunk 1]
```diff
@@ -162,16 +167,28 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
...
   function transformToUrl(field) {
+    if(field.value !== '') {
     field = `https://www.${field.name}.com/${field.value}`
+    }
     return field
   }
+  
+  function visualValidation(field) {
+    const span = document.getElementsByClassName(`edit ${field.name}`)[0]
+    if(field.value) {
```
Comment (vinyll) [inline templates/user/profile.html]: `span.classList.toggle('valiation', field.value)`

**Comment by fuentesloic** [comment]
@newick @vinyll someone to valid this PR ?

**Review by vinyll — APPROVED** [review]

**Pull request merged**